internal/graphql/resolvers/helper.go

```diff
@@ -2,69 +2,168 @@ package resolvers
 
 import (
 	"context"
-	"fmt"
+	"github.com/google/uuid"
 	ent "github.com/mensatt/backend/internal/database/ent"
 	"github.com/mensatt/backend/internal/database/ent/dish"
 	"github.com/mensatt/backend/internal/database/ent/location"
 	"github.com/mensatt/backend/internal/database/ent/occurrence"
 	"github.com/mensatt/backend/internal/database/ent/review"
 	"github.com/mensatt/backend/internal/graphql/models"
 	"io"
+	"net/http"
 )
 
-func (r *mutationResolver) storeImages(tx *ent.Tx, ctx context.Context, review *ent.Review, images []*models.ImageInput) ([]*ent.Image, error) {
-	var imageEntities []*ent.Image
-	for _, image := range images {
-		if image.Image.Size > int64(r.ImageUploader.GetMaxImageSize()) {
-			return nil, fmt.Errorf("image size is too large") // let the caller handle the transaction rollback
-		}
+func (r *mutationResolver) submitImages(uuids []uuid.UUID) []uuid.UUID {
+	var submittedImages []uuid.UUID
 
-		imageBytes, err := io.ReadAll(image.Image.File)
+	for _, imageUUID := range uuids {
```

Comment:
Is there a way to iterate without range? I assume the ignored parameter is the idx?

Reply:
Is this a general go question?